generate.sh

```diff
@@ -0,0 +1,33 @@
+TIMESTAMP=`eval date +%d.%B.%Y`
```

Comment:
Do we really need a shell script to generate them, instead of the Makefile?

If we have a shell-script, it should at least do proper error handling to make it possible to understand if something fails...

Reply:
We should at least enforce shellcheck